Source/DataSources/ModelGraphics.js

```diff
@@ -309,6 +310,13 @@ Object.defineProperties(ModelGraphics.prototype, {
    * @type {Property|undefined}
    */
   clippingPlanes: createPropertyDescriptor("clippingPlanes"),
+
+  /**
+   * A property specifying the {@link ClippingPolygon} used to selectively disable rendering the model.
+   * @memberof ModelGraphics.prototype
+   * @type {Property}
```

Comment:
FYI, should be `@type {Property|undefined}` basically any property that can be undefined has to have the `|undefined` for strict mode TypeScript to work.

Reply:
fixed